docs/dev_guide/contributing/governance.rst

```diff
@@ -1,19 +1,138 @@
+ - Ensure meeting notes are taken by someone present at the meeting and posted to Tripal.info.
+ - Call for votes on TSC recommendations when community voting is required.
+ - Call additional meetings if needed.
+ - Facilitate communication between the TSC and TPMC.
+ - Filling vacant slots on the TSC.
+ - The chair has voting privileges.
+
+TSC Meeting Agenda Items
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Strongly encouraged to be posted to the GitHub Tripal core issue queue as well as to tripal.info, to inform and solicit community comment. TSC meeting agendas will include issues tagged “TSC Next Meeting” on the GitHub Tripal core issue queue. Other agenda items may be added by the TSC chair or members, or by the TPMC. These issues will be closed after the meeting.
+
+Communication and Meetings
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The primary TSC meeting will be held in January of each year, at the Tripal codefest. In-person attendance is strongly encouraged, but a teleconference option will be provided. Each issue on the agenda will be discussed, and if needed, the chair will call for a vote to determine the final recommendation of the TSC. Votes carry based on simple majority. All discussion, votes and objections will be recorded in meeting notes, which will be posted on Tripal.info.
+
+Additional teleconference TSC meetings will be held once per quarter (April, July, October).  These could be held in place of the monthly Tripal User's Meeting to avoid meeting overload.
+
+TSC meetings outside the above schedule may be called by the TSC chair. These will only be called in urgent situations. In less urgent situations, the TSC chair or the TPMC can contact the internal members of the TSC and request a meeting or solicit comments via email, GitHub issue, or Slack.
+
+At any time the TPMC may communicate with members of the TSC with expertise in specific areas if needed to assist in decision making.
+
+Changes to this Document
+--------------------------
 
-- Users: They have downloaded Tripal!  Maybe they even have a site!
-- Contributors: contributing!  Code, comments, discussion, questions, bug reports.
-- Committers: write access to the repository.
-- PMC: Makes **code relevant** decisions.
-- Oversight committee: Makes **policy level** decisions.  This may overlap with PMC, but the idea is the oversight committee includes parties who are **not necessarily coders** and therefore not reviewing Pull requests etc.
+These guidelines are binding to the Tripal Community. If you have comments or questions, please `comment on the Github issue <https://github.com/tripal/tripal/issues/786>`_. Changes to this document will be made after adequate discussion has occurred and the project management committee has voted in favor of the change.
```

Comment:
I would change this to perhaps be "please create an issue for discussion on github or bring your comments/questions to the user meeting", as #786 and the other governance issues are for the formation of the TAC etc, not to serve as a permanent discussion board for it.


```suggestion
These guidelines are binding to the Tripal Community. If you have comments or questions, please `post them as a Github issue <https://github.com/tripal/tripal/issues/new?template=discussion.md>`_ or ask them at the user's meeting. Changes to this document will be made after adequate discussion has occurred and the project management committee has voted in favor of the change.
```